graphrag_sdk/document_loaders/csv.py

```diff
@@ -29,4 +29,4 @@ def load(self) -> Iterator[Document]:
                         ]
                     ]
                 )
-                yield Document(content)
+                yield Document(content, f"{self.path}#{i}")
```

Comment:
this is a very expensive way to split a file into parts.
basically you want to count the number of lines in the file and break it according to `rows_per_document`